example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>9.0</string>
+  <string>11.0</string>
```

Comment:
This and all changes within `example` were automatically made when running an an up-to-date macBook running the latest Xcode, so these changes should be committed. It looks like Flutter automatically tracks the minimum supported SDK (https://developer.apple.com/support/xcode/).